package.json

```diff
@@ -22,16 +22,17 @@
     "test": "grunt test"
   },
   "dependencies": {
+    "amqplib": "^0.5.1",
     "async": "1.5.2",
+    "body-parser": "1.15.0",
     "dateformat": "1.0.12",
     "express": "^4.11.2",
-    "body-parser": "1.15.0",
-    "amqplib": "^0.5.1",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/dcalvoalonso/iotagent-node-lib.git#task/activeAttsNgsi2",
     "logops": "1.0.0-alpha.7",
-    "mqtt": "1.7.0",
-    "request": "^2.69.0",
+    "moment": "^2.20.1",
```

Comment:
Why `moment` has been added? Or, in other words, who is using `moment` module?

Reply:
What `Fixed in ...` means in this context? Did you did any modification regarding `moment` in the cited commit?

Note that my comment is a question. The usual "protocol" is to answer it with a comment, then end with a NTC (= Nothing To Change) if the answer doesn't involves any change (as it seems to be in this case).

Reply:
Sorry was a mistake, NTC